Re-enable credit card option in Desktop+Mobile

moiikana opened the pull request:
- due to some provider issues the credit card option had to be disabled
- it can now be a payment option in the banners and on the donation page again

:warning: 

- [x] Change and deploy the fundraising application to allow for credit card payments again BEFORE pushing these changes to Central Notice

https://phabricator.wikimedia.org/T341452

timEulitz commented:
Very important for https://github.com/wmde/fundraising-banners/pull/148 and https://github.com/wmde/fundraising-banners/pull/147 because they need to load the campaign values in the "new manner".

Follow-up to https://github.com/wmde/fundraising-banners/pull/146

KaiNissen commented on desktop/banner_ctrl.js:
```diff
@@ -68,14 +62,18 @@ const progressBarTextInnerLeft = [
+	campaignProjection,
+	{
+		textInnerLeft: progressBarTextInnerLeft,
+		modifier: 'progress_bar--lateprogress'
```
The modifier needs to be changed for the campaign start.

moiikana committed:
Re-enable credit card option in Desktop+Mobile

- due to some provider issues the credit card option had to be
disabled
- it can now be a payment option in the banners and on the
donation page again

https://phabricator.wikimedia.org/T341452
